THIRD_PARTY.md

```diff
@@ -0,0 +1,68 @@
+### Accompanist
+
+Copyright 2020 The Android Open Source Project
+[https://github.com/google/accompanist](https://github.com/google/accompanist)
+Apache License 2.0
+
+### AndroidX
+
+Copyright 2021 The Android Open Source Project
+[https://github.com/androidx/androidx](https://github.com/androidx/androidx)
+Apache License 2.0
+
+### Coil
+
+Copyright 2023 Coil Contributors
+[https://github.com/coil-kt/coil](https://github.com/coil-kt/coil)
+Apache License 2.0
+
+### Dagger
+
+Copyright 2012 The Dagger Authors
+[https://github.com/google/dagger](https://github.com/google/dagger)
+Apache License 2.0
+
+### Firebase
+
+Copyright 2019 Google LLC
+[https://github.com/firebase/firebase-android-sdk](https://github.com/firebase/firebase-android-sdk)
+Apache License 2.0
+
+### Jekyll
```

Comment:
Jekyll is only a tool. As we don't have any dependency on Jekyll in our code, I am not sure we need to put it here but I might be wrong.